### PR TITLE
Remove "CID charsets"

### PR DESCRIPTION
**Pull request by miguelsousa**
Partial fix for #264

### CODE REVIEW

**Comment by miguelsousa** [comment]
👌

**Comment by miguelsousa** [comment]
Oh fun! Another **branch-passes-pr-fails** case in AppVeyor land :confounded:
This time it's all Python-based tooling, I believe.
https://ci.appveyor.com/project/adobe-type-tools/afdko/build/1.0.279
https://ci.appveyor.com/project/adobe-type-tools/afdko/build/1.0.280

**Comment by miguelsousa** [comment]
@cjchapman I'm done with the changes on this PR

File: afdko/Tools/SharedData/FDKScripts/ProofPDF.py [hunk 1]
```diff
@@ -239,7 +231,7 @@ def CheckEnvironment():
 		logMsg("or the files referenced by the shell script is missing.")
```
Comment (cjchapman) [inline afdko/Tools/SharedData/FDKScripts/ProofPDF.py]: `exe_dir` is undefined

File: afdko/Tools/SharedData/FDKScripts/ProofPDF.py [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+import copy
+import os
+import platform
+import re
+import sys
+import tempfile
+import time
+import traceback
+
+from fontTools.ttLib import TTFont, getTableModule
```
Comment (cjchapman) [inline afdko/Tools/SharedData/FDKScripts/ProofPDF.py]: add `TTLibError` to the imports from `fontTools.ttLib`

File: afdko/Tools/SharedData/FDKScripts/ProofPDF.py [hunk 3]
```diff
@@ -776,15 +734,14 @@ def openFile(path, txPath):
...
 
 		if len(data) < 10:
 			raise FontError("Error: font file was zero size: may be a resource fork font, which this program does not process. <%s>." % path)
 		if (data[:4] == "OTTO") or (data[:4] == "true") or (data[:4] == "\0\1\0\0"): # it is an OTF/TTF font, can process file directly
 			try:
-				ttFont = ttLib.TTFont(path)
+				ttFont = TTFont(path)
 			except (IOError, OSError):
 				raise FontError("Error opening or reading from font file <%s>." % path)
 			except TTLibError:
```
Comment (cjchapman) [inline afdko/Tools/SharedData/FDKScripts/ProofPDF.py]: `TTLibError` is undefined -- see comment at top of file about need to import it from `fontTools.ttLib`

File: afdko/Tools/SharedData/FDKScripts/FDKUtils.py [hunk 4]
```diff
@@ -9,14 +9,13 @@
 import traceback
```
Comment (cjchapman) [inline afdko/Tools/SharedData/FDKScripts/FDKUtils.py]: line 30: change `fdkScriptsDir = None` to `fdkSharedDataDir = None`

lines 167 - 176: if this is still needed, then change `import FDK` to `import afdko`

**Comment by miguelsousa** [comment]
Dunno what's going on with the Linux build

**Comment by cjchapman** [comment]
I'm going to try to sort out the Travis/Linux issue in another branch/PR.

**Comment by cjchapman** [comment]
...and now Travis/Linux is magically working. <sigh>

**Comment by cjchapman** [comment]
@miguelsousa I approve the changes. I leave it to you to squash-and-merge.

**Pull request merged**